src/ResourceManager/Profile/Commands.Profile/StartupScripts/AzError.ps1

```diff
@@ -15,6 +15,10 @@
             Remove-Item -Path $pathToInstallationChecks -ErrorAction Stop
         }
 
+        $installationChecksDirectory = Split-Path -Path $pathToInstallationChecks -Parent -ErrorAction Stop
+        if (-not (Test-Path $installationChecksDirectory -ErrorAction Ignore)) {
```

Comment:
I think we should only write the file if the directory exists - we essentiallyshouldn't write errors in headless scenarios, I think, i.e.:

```powershell
$installationChecksDirectory = Split-Path -Path $pathToInstallationChecks -Parent -ErrorAction Stop
if (Test-Path -Path $installationChecksDirectory -ErrorAction Ignore) {
    New-Item -Path $pathToInstallationChecks -ErrorAction Stop -ItemType File -Value ($installationchecks | ConvertTo-Json -ErrorAction Stop)
}
```